<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DPrjProp</class>
 <widget class="QDialog" name="DPrjProp">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>405</width>
    <height>323</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>381</width>
     <height>121</height>
    </rect>
   </property>
   <property name="title">
    <string>Debug Mode</string>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QRadioButton" name="mDebugSimulator">
      <property name="text">
       <string>Simulator</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="mDebugUsb">
      <property name="text">
       <string>USB controller</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="mDebugRemote">
      <property name="text">
       <string>Remote on local net</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="mDebugBridge">
      <property name="text">
       <string>Remote over Internet through bridge</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Main script:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mMainScript">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>140</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mSelectScript">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>140</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Select...</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>IP:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mIp">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Port:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mPort">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>200</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Controller name:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="mControllerName">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>230</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="mOk">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="mCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Cancel</string>
   </property>
   <property name="autoDefault">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="mControllerPassw">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>260</y>
     <width>191</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Controller passw:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
